website/docs/reference/deploy/configuring-unleash.md

```diff
@@ -180,6 +180,20 @@ const start = async () => {
 start();
 ```
 
+### Segment limits {#segments}
+
+:::caution
+
+Changing segment limits could have adverse impacts on the performance of Unleash SDKs and cause network congestion. Think twice before changing these values.
```

Comment:
```suggestion
Changing segment limits could have an adverse impact on the performance of Unleash SDKs and cause network congestion. Think twice before changing these values.
```
or maybe 
```suggestion
Changing segment limits could have a negative impact on the performance of Unleash SDKs and cause network congestion. Think twice before changing these values.
```
Which I heard more frequently

Reply:
Sure! Negative is probably clearer than adverse, so that's good 😄